crates/evm/src/evm/system_contracts/src/Bridge.sol

```diff
@@ -11,18 +14,35 @@ import "./L1BlockHashList.sol";
 /// @author Citrea
 
 contract Bridge is MerkleTree, Ownable {
-    // TODO: Update this to be the actual address of the L1BlockHashList contract
-    L1BlockHashList public constant BLOCK_HASH_LIST = L1BlockHashList(address(0xdeaDDeADDEaDdeaDdEAddEADDEAdDeadDEADDEaD)); 
+    using BTCUtils for bytes;
+    using BytesLib for bytes;
+
+    struct DepositParams {
+        bytes4 version;
+        bytes2 flag;
+        bytes vin;
+        bytes vout;
+        bytes witness;
+        bytes4 locktime;
+        bytes intermediate_nodes;
+        bytes block_header;
```

Comment:
We don't need this since we have access to witness merkle root with the block height